test/unit/proxy_test.rb

```diff
@@ -478,20 +478,13 @@ def test_apicast_configuration_driven
     end
   end
 
+ #TODO: remove the test when the ApicastV1DeploymentService class is removed
```

Comment:
```suggestion
```
Rather than removing the test, it should say remove [line 487](https://github.com/3scale/porta/pull/2495/files#diff-cc3f6466d2246468906e67e3164b6339cd96c6e611daa30aad5eb2cfe82e15b7R487):
```ruby
::ApicastV1DeploymentService.any_instance.expects(:deploy).with(proxy).returns(true).never
```